.development.env

```diff
@@ -15,7 +15,6 @@ EQ_REDIS_HOST=localhost
 EQ_REDIS_PORT=6379
 EQ_SECRETS_FILE=dev-secrets.yml
 EQ_KEYS_FILE=dev-keys.yml
-EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE=2021-04-28T14:00:00+00:00
```

Comment:
Removed the env vars here but they are still expected in the deploy_app.yaml file. Will also need to update the pipelines and terraform and test deploying runner with the vars removed.

Reply:
Updated and Tested. New PRs have been added to the card:

- [Pipelines PR](https://github.com/ONSdigital/eq-pipelines/pull/658)
- [Terraform GCP Runner PR](https://github.com/ONSdigital/eq-terraform-gcp-runner/pull/41)